packages/module/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "gravity-nuxt",
```

```suggestion
  "name": "@gravity-vue/nuxt",
```
Не забудь обновить везде где используется